### PR TITLE
Add aws card to missing aws docs

### PR DESCRIPTION
**Pull request by reshmabidikar**
Changes for https://github.com/killbill/technical-support/issues/173.

### CODE REVIEW

File: userguide/tutorials/using-https.adoc [hunk 1]
```diff
@@ -1,5 +1,7 @@
 = Using HTTPS
```
Comment (pierre) [inline userguide/tutorials/using-https.adoc:1]: Is this file still in use? I couldn't find where it's linked on the left? Also, is it different than https://docs.killbill.io/latest/how-to-add-a-certificate-using-ACM ?

Maybe it's related to the clean-up work from Jim (https://github.com/killbill/killbill-docs/pull/555)? Would be good to close the loop on that PR too.

File: userguide/tutorials/using-ses-with-aws.adoc [hunk 2]
```diff
@@ -1,5 +1,7 @@
 = Using SES with AWS
```
Comment (pierre) [inline userguide/tutorials/using-ses-with-aws.adoc:1]: Should we move all AWS related files under the `userguide/aws` directory?

**Review by pierre — APPROVED** [review]
Just a couple of questions.

File: userguide/tutorials/using-ses-with-aws.adoc (hunk 2 above)
Comment (reshmabidikar) [reply userguide/tutorials/using-ses-with-aws.adoc:1]: Yes, that makes sense. I will do this in a separate PR.

File: userguide/tutorials/using-https.adoc (hunk 1 above)
Comment (reshmabidikar) [reply userguide/tutorials/using-https.adoc:1]: > Maybe it's related to the clean-up work from Jim (https://github.com/killbill/killbill-docs/pull/555)

Yes, it is possible, I'll double-check if this file is still in use.

**Pull request merged**